app/scripts/bootstrap.js

```diff
@@ -9,11 +9,17 @@ $(function () {
     window.SwaggerEditor.$defaults = embeddedDefaults;
     angular.bootstrap(window.document, ['SwaggerEditor']);
   } else {
-    $.getJSON('/config/defaults.json').done(function (resp) {
+    var rootPath = '';
+    if (window.location.pathname.lastIndexOf('/') !==
```

Comment:
```javascript
$.getJSON('/config/defaults.json')
```
This make a request base on the url root. if we deploy the dist folder to sub path like http://domain.com/editor/index.html. Then it get the defaults.json by the url http://domain.com/config/defaults.json so throw a 404. 

the window.location.pathname would be end with '/' or 'html' or a dirname like 'editor'. When it end with '/', it dosen't need to add a root path,just change the '/config/defaults.json' to './config/defaults.json' will be ok. When it end with 'html' or a dirname like 'editor' , the root path is the parent directory, so rootPath = pathname + '/../' . This logic fixes the defaults.json and the examples yaml 404 error, when deploy it in a sub folder or reverse proxy by nginx.